github/actions_secrets.go

```diff
@@ -101,12 +127,14 @@ type EncryptedSecret struct {
 	Name           string `json:"-"`
 	KeyID          string `json:"key_id"`
 	EncryptedValue string `json:"encrypted_value"`
+	Visibility     string `json:"visibility,omitempty"`
+	SelectedRepositoryIDs
```

Comment:
Then this would become:

```
SelectedRepositoryIDs SelectedRepoIDs `json:"selected_repository_ids,omitempty"`
```

Note the long field name to correspond to the long JSON parameter name but the shorter type name.